salepoint/resources/views/partners/profile.blade.php

```diff
@@ -97,7 +97,13 @@
                            @endif
                            disabled/>
                 </td>
+                <td valing="top">
```

Comment:
Esto le dije a mario que te dijera que lo quitaras we... solo lo hicimos de prueba